frontend/src/layouts/ContentLayout.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { CssBaseline, Paper } from '@mui/material';
+
+const styles = {
+  main: {
+    width: 'auto',
+    display: 'block', // Fix IE 11 issue.
+    marginLeft: 100,
+    marginRight: 100,
+  },
+  paper: {
+    marginTop: 3,
+    paddingBottom: 20,
```

Comment:
this field is overrided by `p: 2` and is redundant. Should be removed or modified.

Reply:
`p: 2` is removed in commit ae7e0e1